packages/salesforcedx-vscode-core/package.json

```diff
@@ -707,11 +707,10 @@
           "default": false,
           "description": "%internal_dev_description%"
         },
-        "salesforcedx-vscode-core.show-org-browser": {
-          "type": [
-            "boolean"
-          ],
-          "default": false
+        "salesforcedx-vscode-core.orgBrowser.enabled": {
```

Comment:
Calling it `orgBrowser.enabled` gives the user that impression that the setting is already enabled when seeing it in the UI. At least we've had users being confused about it in the past.

Reply:
VS Code uses this convention throughout their settings. I think it was confusing for us before because we weren't surfacing the setting through the UI to see the state.

![Screen Shot 2019-07-23 at 9 25 09 AM](https://user-images.githubusercontent.com/10779873/61729323-c9105300-ad2b-11e9-8f07-4a2b141c7638.png)

Reply:
I see what you mean, then let's keep it as it is right now.